invenio_jobs/services/schema.py

```diff
@@ -130,7 +131,7 @@ class Meta:
         validate=LazyOneOf(choices=lambda: current_jobs.queues.keys()),
         load_default=lambda: current_jobs.default_queue,
     )
-    default_args = fields.Dict(load_default=dict)
+    default_args = fields.Dict(load_default=dict, allow_none=True)
```

Comment:
Wondering if this is an intentional change since it's not related to time zones nor `RunStatusEnum`.

Reply:
yes this is intentional, and I probably should've added it as a separate commit! essentially alex/yash decided that default_args can be none